kuma/static/js/components/payments/focus-visible.js

```diff
@@ -10,40 +10,45 @@
         this.removeEventListener('focusout', removeFocusVisible);
     }
 
-    // When a keyup event is triggered on the contribute form,
-    contributeForm.addEventListener('keyup', function(event) {
-        var currentTarget = event.target;
-        var targetType = currentTarget.type;
+    if (contributeForm) {
```

Comment:
Given that the body of this if statement is basically then entire file, consider up at line 4 just adding `if (!contributeForm) return;`

Reply:
Doh! only saw this after the merge. That is much better. Additional PR coming your way.